presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -0,0 +1,119 @@
+package io.prestosql.plugin.hive.metastore.cache;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.io.Closer;
+import com.google.common.io.Files;
+import com.google.common.io.MoreFiles;
+import io.prestosql.Session;
+import io.prestosql.plugin.hive.HiveHadoop2Plugin;
+import io.prestosql.spi.security.Identity;
+import io.prestosql.spi.security.SelectedRole;
+import io.prestosql.testing.DistributedQueryRunner;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static com.google.common.io.RecursiveDeleteOption.ALLOW_INSECURE;
+import static io.prestosql.spi.security.SelectedRole.Type.ROLE;
+import static io.prestosql.testing.TestingSession.testSessionBuilder;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@SuppressWarnings("UnstableApiUsage")   // for Closer
+public class TestCachingHiveMetastoreWithQueryRunner
```

Comment:
extend `io.prestosql.testing.AbstractTestQueryFramework`

Reply:
I don't need any functionality from this base class

also as we discussed internally, reusing test code via inheritance does more harm than good

Reply:
Just FYI there is io.prestosql.sql.query.QueryAssertions

Reply:
I just need to know if query passes or fails in this test, so no assertion utility class is needed for me, just regular method calls